packages/fx-core/src/common/constants.ts

```diff
@@ -20,3 +20,7 @@ export class PluginDisplayName {
 export class FeatureFlagName {
   static readonly MultiEnv = "TEAMSFX_MULTI_ENV";
 }
+
+export class ProjectVersion {
+  static readonly V1 = "v1";
```

Comment:
Is the project version config only added for v1 project. For v2, shall we need to also add this config for consistency?

Reply:
The `projectVersion` is similar to the property `version` in `SolutionSettings`.
Rename it to `migrateFromV1`.

PS: `Version` in `SolutionSettings` means the solution plugin version.